### ISSUE TITLE
nbcsports.com detects adblock

### ISSUE THREAD

**Issue by jbjorgen** [reek/anti-adblock-killer#3591]
When attempting to view any video on nbcsports.com, the following message appears where the video would normally display:  

PLEASE TURN OFF YOUR AD BLOCKER
This video is unavailable if an ad blocker is running. Please disable and reload the page to start the video. 

<img width="915" alt="screen shot 2017-09-18 at 1 08 17 am" src="https://user-images.githubusercontent.com/46091/30530625-ec4fb710-9c0d-11e7-8f89-0d44b51879a6.png">

Firefox with Greasemonkey and Adblock

**Comment by uBlock-user** [comment]
`@@/ads.js$domain=nbcsports.com`

uBO - `||hdliveextra-a.akamaihd.net/*/ads.js$script,redirect=noopjs`

**Comment by jbjorgen** [comment]
Excuse my ignorance, but I'm assuming this needs to be inserted into a script at some point.  Is there a specific line number or line I should look for?

I tried updating my filter list, but it didn't seem to have any affect.

**Comment by uBlock-user** [comment]
No, this is to be added to your Custom Filters OR My Filters page.

**Comment by jbjorgen** [comment]
Magic!  Thanks a lot...that works great!
